docs/usage_guide.rst

```diff
@@ -102,7 +102,14 @@ You can create a KML object by reading a KML file as a string
 
     # Start by importing the kml module
     >>> from fastkml import kml
-
+    
+    #Read file into string and convert to UTF-8
+    >>> with open(kml_file, 'r') as myfile:
+    ...     data=myfile.read().replace('\n', '')
```

Comment:
why?

Reply:
Sure newline replacements are unnecessary. It was just some other code for kml reading I had lying around.